### DISCUSSION TITLE
I have 11 discussions answered yet no badge.

### DISCUSSION THREAD

**Discussion by devkennyy** [community/community#20943] [Other Feature Feedback, Questions, & Ideas] [answered]
You can see on my profile, that I have 11 discussions answered. None of these are marked as answer by myself yet I haven't received the badge. [Example 1](https://github.com/devkennyy/rungeon/discussions/107), [Example 2](https://github.com/monkeytypegame/monkeytype/discussions/2134). These should at least provide me with the minimum.

**Comment by aaronlamz** [comment, marked as answer]
Hi, to get the Galaxy Brain achievement badge, you need to fulfill the following criteria:

* Discussions should be in a public repository
* Get marked as answers to 2 or more questions
* The questions cannot be asked by you. In short, you cannot answer your own questions, those do not count.
* There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.
* Reference: #18454
Hope these steps help you in achieving your badge! Good luck :)

More details here:
https://github.com/Schweinepriester/github-profile-achievements

**Reply by devkennyy** [reply]
The repository I had marked my own answers for has been deleted. Does this mean I can't ever get the badge?

**Reply by aaronlamz** [reply]
I'm not quite sure. You can  unmark all your answers to your questions and try again.

<img width="925" alt="image" src="https://user-images.githubusercontent.com/3964466/180462690-0031edb1-9c70-4ce3-acc0-2825de859f99.png">

more detail here #18454
Hope this helps.

**Reply by devkennyy** [reply]
Some of my answers are on deleted repositories, yikes but this is still the right answer. Thanks!

**Comment by jfullstackdev** [comment]
it's not the badge that you think, rather this one

![image](https://user-images.githubusercontent.com/47092464/179991350-5dc7e6c8-d409-4cba-a27c-f61e35e9e7b3.png)

at the bottom, 11 discussions answered

**Reply by MrSarno** [reply]
This is incorrect; there is also a badge available.

You can see an example of someone with the badge here;

<img width="363" alt="Screenshot 2022-07-22 at 16 46 27" src="https://user-images.githubusercontent.com/32073555/180475884-602134b5-afbb-43f2-a6c4-2ee7496f9796.png">

That same person has a list of known GitHub profile badges, which you can find [here](https://github.com/Schweinepriester/github-profile-achievements).

**Comment by radar-blip** [comment]
It might be that your answers have to be within a certain time period (like 30 days), but im not sure. I've seen others with more answers and no achievement badge.

**Reply by devkennyy** [reply]
All of these were after the badge's release, weird.
